StripeConnect/StripeConnect/Source/Helpers/Appearance+Encoding.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  Appearance+Encoding.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 9/4/24.
+//
+
+import StripeCore
+import UIKit
+
+typealias Appearance = EmbeddedComponentManager.Appearance
+
+extension Appearance {
+    func asDictionary(traitCollection: UITraitCollection) -> [String: String] {
+        var dict: [String: String] = [:]
+        
+        dict.mergeAssertingOnOverwrites(colors.asDictionary(traitCollection: traitCollection))
+        dict.mergeAssertingOnOverwrites(typography.asDictionary())
+        dict.mergeAssertingOnOverwrites(buttonPrimary.asDictionary(keyPrefix: "buttonPrimary"))
+        dict.mergeAssertingOnOverwrites(buttonSecondary.asDictionary(keyPrefix: "buttonSecondary"))
+        dict.mergeAssertingOnOverwrites(badgeNeutral.asDictionary(keyPrefix: "badgeNeutral"))
+        dict.mergeAssertingOnOverwrites(badgeSuccess.asDictionary(keyPrefix: "badgeSuccess"))
+        dict.mergeAssertingOnOverwrites(badgeWarning.asDictionary(keyPrefix: "badgeWarning"))
+        dict.mergeAssertingOnOverwrites(badgeDanger.asDictionary(keyPrefix: "badgeDanger"))
+        dict.mergeAssertingOnOverwrites(cornerRadius.asDictionary())
+        
+        dict["spacingUnit"] = spacingUnit?.pxString
+        
+        return dict
+    }
+}
+
+extension Appearance.CornerRadius {
+    private var mappings: [String: WritableKeyPath<Self, CGFloat?>] {
```

Comment:
Could this use `KeyPath` instead of `WritableKeyPath`? It looks like we're only reading from and never writing to the property